pikachu_education/lib/components/positive_button.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/material.dart';
+import 'package:pikachu_education/utils/management_color.dart';
+
+class PositiveButtonCustom extends StatelessWidget {
+  const PositiveButtonCustom(
+      {super.key, required this.nameButton, required this.onPressed,required this.stateLoading});
+
+  final VoidCallback onPressed;
+  final String nameButton;
+  final bool stateLoading;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(horizontal: 20),
+      child: ClipRRect(
+        borderRadius: BorderRadius.circular(10),
+        child: Container(
+          width: MediaQuery.of(context).size.width,
+          height: 60,
+          color: Colors.transparent,
```

Comment:
Don't need to use the `ClipRRect` to make `BorderRadius`. Just simply by using the `decoration` property of `Container`.
Notice that, when you use the `decoration` you have to move the `color` into the `decoration`.

Reply:
Dear @congthanhng please check Re-edit